homeassistant/components/climate/services.yaml

```diff
@@ -94,3 +94,27 @@ set_swing_mode:
     swing_mode:
     description: New value of swing mode
     example: 1
+
+ecobee_set_fan_min_on_time:
```

Comment:
If you put these here won't they show up on any/all climate components? If this is intended for ecobee only it shouldn't go here.